### PR TITLE
adding more instructions for group mentoring coordinators 

### PR DESCRIPTION
**Pull request by parispittman**
adding clarity and steps for group mentoring coordinators 

/area mentorship-planning
/sig contributor-experience

### CODE REVIEW

File: mentoring/processes/group-mentoring.md [hunk 1]
```diff
@@ -54,24 +55,32 @@ Examples:
...
+ - issue #2: structure of sessions and assign guest speakers 
+ start with a hackmd to coordinate with mentors and then set issue
+1. ContribEx Coordinator(s) posts issue #1 to the following places:
+  - SIG list first and then k-dev mailing lists with timeboxes for responses
   - #diversity, #kubernetes-contributors, #sig-[name], and other slack channels
-3. Coordinator arranges a doodle for the kick off meeting and days/times that 
-would be best for the mentor and group for slack standups and touch points. Once conscensus is reached, coordinator schedules it.
-4. Mentor creates template that serves as a reference guide for the group. See
+3. Slack stand ups happen on the day of the SIG meetings but the 30-min
+bi-weekly meetings will need to be arranged. Coorindator should send out a
```
Comment (krmayankk) [inline mentoring/processes/group-mentoring.md:69]: nit: Coordinator

File: mentoring/processes/group-mentoring.md [hunk 2]
```diff
@@ -54,24 +55,32 @@ Examples:
...
+1. ContribEx Coordinator(s) posts issue #1 to the following places:
+  - SIG list first and then k-dev mailing lists with timeboxes for responses
   - #diversity, #kubernetes-contributors, #sig-[name], and other slack channels
-3. Coordinator arranges a doodle for the kick off meeting and days/times that 
-would be best for the mentor and group for slack standups and touch points. Once conscensus is reached, coordinator schedules it.
-4. Mentor creates template that serves as a reference guide for the group. See
+3. Slack stand ups happen on the day of the SIG meetings but the 30-min
+bi-weekly meetings will need to be arranged. Coorindator should send out a 
+doodle to the selected group with a few times that work for the mentors.
+4. Coorindator and mentors work together on the three-month plan template.
```
Comment (krmayankk) [inline mentoring/processes/group-mentoring.md:71]: same

**Comment by jberkus** [comment]
oops, didn't realize this wasn't merged.

/lgtm

**Pull request merged**